src/messages/manifestjson.js

```diff
@@ -888,3 +888,13 @@ export const MISSING_ADDON_ID = {
     future. See https://mzl.la/3PLZYdo for more information.`),
   file: MANIFEST_JSON,
 };
+
+export const BACKGROUND_SERVICE_WORKER = {
+  code: 'BACKGROUND_SERVICE_WORKER',
+  message: i18n._(`"/background/service_worker" is not supported.`),
+  description: i18n._(`The "background" property must contain either "scripts" 
+    or "page". See 
```

Comment:
Instead of offering two options, let's offer "scripts" as a fallback. The MDN docs provides more context:

```suggestion
  description: i18n._(`"background.scripts" must be specified as a fallback for
    "background.service_worker". See
```